examples/gzip/service.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+from flask.helpers import send_file
+from flask_compress import Compress
+
+app = Flask(__name__)
+app.config["COMPRESS_ALGORITHM"] = 'gzip'
+Compress(app)
```

Comment:
this doesnt want  to be compressed - envoy should be doing the compression

Reply:
I test, seems can not, if flask do not return compressed contents, Envoy would not do it :( 
Maybe cc any experts here, while I do not familiar with them.

Reply:
if its being compressed here then envoy is not compressing the file

Reply:
I remove compress code here and do not get compressed contents.

Reply:
check the content-type maybe ?

Reply:
It is always `application/json`.

With `curl` option `-w '%{size_download}\n'`, I confirm it.

Reply:
whatever the reason - the need to compress here means the example isnt working - flask is compressing not envoy

Reply:
I think original flask do not meet [requirement](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/gzip_filter#how-it-works).

Reply:
not sure that is the case - it *should* work - i can check further if you dont manage to figure it out

as i said above tho - this is currently demonstrating flask's compression not envoy's

Reply:
After change a function, it works.